docs/Installation-Guide.rst

```diff
@@ -558,6 +558,35 @@ Docker
 
 Refer to `GPU Docker folder <https://github.com/microsoft/LightGBM/tree/master/docker/gpu>`__.
 
+Build CUDA Version (Experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `original GPU version <#build-gpu-version>`__ of LightGBM is based on OpenCL. This version is a separate implementation and requires NVIDIA graphics card with compute capability 6.0 and higher.
+
+This version is very experimental and we suggest using it only when it is impossible to use OpenCL version, for instance, on IBM POWER microprocessors.
+
+**Note**: only Linux is supported, other operating systems are not supported yet.
+
+Linux
+^^^^^
+
+On Linux a CUDA version of LightGBM can be built using **CUDA**, **CMake** and **gcc** or **Clang**.
+
+The following dependencies should be installed before compilation:
+
+-  **CUDA** libraries.
+
+-  **CMake** 3.16 or later.
+
+To build LightGBM CUDA version, run the following commands:
+
+.. code::
+
+  git clone --recursive https://github.com/microsoft/LightGBM ; cd LightGBM
+  mkdir build ; cd build
+  cmake -DUSE_CUDA=1 ..
+  make -j4
```

Comment:
I have a slight preference for putting each command on its own line, instead of grouping them with `;`. That makes it a little easier to understand at first glance, in my opinion.

```suggestion
  git clone --recursive https://github.com/microsoft/LightGBM
  cd LightGBM
  mkdir build
  cd build
  cmake -DUSE_CUDA=1 ..
  make -j4
```

What do you think? If you disagree, it's ok to leave it as-is. It's not worth holding up the PR.

Reply:
I totally agree! But could it be done in a separate PR to change all occurrences of such commands across the doc?

Reply:
oh sure, no problem. I didn't know there were other places that do this. It can be another PR, and not urgent at all

Reply:
Thanks! I'll prepare a new PR right after merging this one.